packages/server/src/app.js

```diff
@@ -13,12 +13,12 @@ import errorMiddleware from './middleware/error';
 
 const app = express();
 
+app.use(cookiesMiddleware());
```

Comment:
We cannot execute this before beforeware. Beforeware must be executed first, so we need to find other solution.